### PR TITLE
Add intel 1

### PR DESCRIPTION
**Pull request by schettn**
**Checklist:**

 
 - [X] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [X] Have you updated or added documentation for the change?
 - [X] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [X] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- #8 UI and App logic are not independent
- #9 It is not possible to switch between dummy data and real data
- #10 There is no way to store data
- #11 There is no dummy data

**What is the new behavior (if this is a feature change)?**
- UI and App logic are separated
- It is possible to switch between dummy data and real data
- Data can be stored in a browser side database now
- The database is filled with dummy data

### CODE REVIEW

File: src/intel/IntelDummy/index.js [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+        ],
+        20
+    );
+    create_dummy(
+        insert.repository,
+        [
+            faker.internet.avatar,
+            faker.internet.userName,
+            function () {
+                return random(count("member"));
```
Comment (schettn) [inline src/intel/IntelDummy/index.js]: This function should be removed. It is not expected in the insert statement

File: src/intel/IntelDummy/index.js [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+            function () {
+                return random(count("organization"));
+            },
+            function () {
+                return random(count("member"));
+            }
+        ],
+        20
+    );
+};
```
Comment (schettn) [inline src/intel/IntelDummy/index.js]: Repositories are not linked with platforms.
Repositories are not filled with members.

File: src/App.js [hunk 3]
```diff
@@ -17,7 +21,6 @@ import {
 */
 // Routes
 import Routes from "./Routes";
-
```
Comment (kleberbaum) [inline src/App.js]: There should be a newline after the imports.

File: src/intel/Database/Statements/Insert/index.js [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    createdAt,
+    location,
+    statusMessage,
+    statusEmojiHTML
+)
+VALUES (?,?,?,?,?,?,?,?,?,?,?,?);
+`;
+
+export const organization = `
+    INSERT INTO organization(
```
Comment (kleberbaum) [inline src/intel/Database/Statements/Insert/index.js]: There shouldn't be a indent at the start of a string.

**Pull request merged**